src/Editor/GUI/Editors/EntityEditor.cpp

```diff
@@ -473,21 +473,45 @@ void EntityEditor::ScriptEditor(Component::Script* script) {
                 else if (typeId == script->instance->GetEngine()->GetTypeIdByDecl("Entity@")) {
 
                     // Find method to call.
-                    std::string entityName = script->instance->GetPropertyName(n);
+                    std::string propertyName = script->instance->GetPropertyName(n);
```

Comment:
A variable with the same value and name already exists in the beginning of the for loop. You may want to consider removing this line altogether.